fiftyone/utils/data/importers.py

```diff
@@ -676,7 +676,11 @@ def _load_data_map(data_path, ignore_exts=False, recursive=False):
                 )
 
             data_map = etas.read_json(data_path)
-            return {to_uuid(k): v for k, v in data_map.items()}
+            data_root = os.path.dirname(data_path)
+            return {
+                to_uuid(k): os.path.join(data_root, v)
```

Comment:
This didn't need to be implemented as an optional `abspath=True` behavior: it's good to always interpret relative paths w.r.t. the location of the manifest

Note that `os.path.join("/this/is/not/used", "/a/b/c.txt") == "/a/b/c.txt"`, so extra conditional logic to handle whether `v` is absolute or not is not necessary.